css-alpha-channel/index.html

```diff
@@ -0,0 +1,79 @@
+---
+feature_name: Alpha Channel Support in CSS Color Parsing
+chrome_version: 52
+feature_id: 5685348285808640
+---
+
+<h3>Background</h3>
+<p>Starting with Chrome 52, CSS rules now accept an alpha (transparency) channel for color values. Do this by specifying a fourth number in contexts that take an RGB value. Alpha channel support takes several forms.</p>
```

Comment:
Link to https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#rgba somewhere from within this paragraph.